src/go/rpk/go.mod

```diff
@@ -59,4 +59,4 @@ require (
 	mvdan.cc/sh/v3 v3.1.1 // indirect
 )
 
-go 1.13
+go 1.15
```

Comment:
This should definitely be in its own PR, since (apart from any testing involved) it has to be coordinated with internal tooling/build changes to that depend on the go version (and will break when the quotes test changes) .

Reply:
Agreed, I don't have access to the internal tooling to help push this through properly. Thanks for reviewing @0xdiba 🙇